### DISCUSSION TITLE
Wildcard search (revisit)

### DISCUSSION THREAD

**Discussion by PieterL75** [netbox-community/netbox#8254] [Ideas]
Is there a way to search using wildcards ?
What I'm trying to do is search for 'server*.domain' and get a result like
- server001.domain
- server004.domain

there was a previous discussion about it (https://github.com/netbox-community/netbox/issues/4658), but looks like it never got traction at django-filters...

This is really a missing feature for us in netbox. In our previous cmdb, we used that type of lookups quite a lot

Pieter

**Comment by corriganalan** [comment]
another vote for wildcard searching
